### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by secureamd**

### CODE REVIEW

**Comment by sanishlodha** [comment]
check this out

https://github.com/sanishlodha/helloworld/pull/1/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R4-R22

File: README.md [hunk 1]
```diff
@@ -1 +1,24 @@
-# helloworld
\ No newline at end of file
+# helloworld
+
+```
+export const createContent = (languages) => {
```
Comment (secureamd) [inline README.md:4]: should i remove this export?

Comment (sanishlodha) [reply README.md:4]: yes!

**Review by sanishlodha — CHANGES_REQUESTED** [review]

**Review by sanishlodha — APPROVED** [review]

**Review by sanishlodha — APPROVED** [review]

**Review by sanishlodha — APPROVED** [review]

**Review by sanishlodha — APPROVED** [review]

**Review by secureamd — COMMENTED** [review]
done

**Pull request merged**